.github/pre-commit-config.yaml

```diff
@@ -32,15 +32,32 @@ repos:
 
   # Third party pre-commit hooks
   - repo: https://github.com/koalaman/shellcheck-precommit
-    rev: v0.9.0
+    rev: v0.10.0
     hooks:
       - id: shellcheck
         args: ["--severity=warning"]
         exclude: ^.*/gradlew$
   - repo: https://github.com/pecigonzalo/pre-commit-shfmt
-    rev: v2.1.0
+    rev: v2.2.0
     hooks:
       - id: shell-fmt
         args:
           - -i
           - "4"
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: "v4.0.0-alpha.8"
+    hooks:
+      - id: prettier
+        # Exclude non-website files for now
```

Comment:
@AndrewDryga Only enabled for website, now that we have more people working on docs and will be making changes I think it's good to have some lints enabled.